README.md

```diff
@@ -80,7 +80,7 @@ my_project/                 <-- project's root
         gallery/
             photo1.jpg      <-- accessible at http://localhost:8080/gallery/photo1.jpg
     vendor/
-        autoload.php        <-- not accessible from web, but PHP can: require(__DIR__ . '/../vendor/autoload.php')
+        autoload.php        <-- not accessible from web, but PHP can: require __DIR__ . '/../vendor/autoload.php'
```

Comment:
To `require()` je tam úmyslně, protože bez závorek je malinko hůř rozlišitelné, že jde o kód, nikoliv o text, ačkoliv jinak souhlasím, že je to nepreferovaný zápis příkazu.